webapp/src/immersivereader.tsx

```diff
@@ -190,10 +190,10 @@ function getTokenAsync(): Promise<ImmersiveReaderToken> {
     const cachedToken: ImmersiveReaderToken = pxt.Util.jsonTryParse(storedTokenString);
 
     if (!cachedToken || (Date.now() / 1000 > cachedToken.expiration)) {
-        return pxt.Cloud.privateGetAsync("immreader").then(
+        return pxt.Util.requestAsync({ url: "https://makecode.com/api/immreader", method: "GET" }).then(
```

Comment:
This needs to be relative path otherwise arcade.staging.pxt.io is going to point to production.

Reply:
Agreed it needs to be relative. For staging and also localhost development. Maybe condition this behind an isElectron check.

Reply:
👍 I changed it to use the forceliveendpoint parameter (found from https://github.com/microsoft/pxt/pull/4209/commits/8d6ce291ef2641107fdfbd3d9c74436087e4d953 )